vue/components/ui/molecules/table/table.custom.stories.js

```diff
@@ -59,5 +61,30 @@ storiesOf("Molecules", module).add("Table Custom", () => ({
                 </tr>
             </template>
         </table-ripe>
+        <h2>Table 2 - Column override:</h2>
+        <table-ripe
+            class="table"
+            v-bind:columns="mockColumns"
+            v-bind:items="mockItems"

```

Comment:
rows instead of items

Reply:
I'll rename to `data` on a different PR, it's too big of a change to mix with this one.